### PR TITLE
Survey Templates 

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

File: gem/settings/base.py [hunk 1]
```diff
@@ -133,7 +133,7 @@
 
 # We have multiple layouts: use `base`, `malawi` or `springster`
 # to switch between them.
-SITE_LAYOUT_BASE = environ.get('SITE_LAYOUT_BASE', 'base')
+SITE_LAYOUT_BASE = environ.get('SITE_LAYOUT_BASE', 'springster')
```
Comment (nathanbegbie) [inline gem/settings/base.py]: Please remember to revert this before committing. I think the other option is to run `export SITE_LAYOUT_2='springster'` in your shell before running the server.

**Comment by Mitso** [comment]
@projectmushroom @nathanbegbie please review 

screen shot 2017-11-15 at 3 09 10 pm
screen shot 2017-11-15 at 2 53 08 pm

**Comment by nathanbegbie** [comment]
@Mitso could we please get a screenshot of how it looks on the feature phone version?

**Comment by Mitso** [comment]
@nathanbegbie @projectmushroom 
These don't tend to be repsonsive on feature phones - I decided to rather strip them out.

screen shot 2017-11-15 at 3 29 25 pm

**Pull request merged**